@navikt/core/react/src/pagination/Pagination.tsx

```diff
@@ -47,6 +48,13 @@ export interface PaginationProps extends React.HTMLAttributes<HTMLElement> {
    * @default (item: PaginationItemProps) => <PaginationItem {...item} />
    */
   renderItem?: (item: PaginationItemProps) => ReturnType<React.FC>;
+  /**
+   * Pagination-heading. We recommend adding heading instead of `aria-label` to help assistive technologies with an extra navigation-stop.
```

Comment:
Tried to do some research to find out when to use hyphen between nouns. It was hard to find/understand any rules about this, but my impression is that hyphen should not be used most of the time.

Found this: "Hyphenate expressions in which each element has equal status, and expressions in which the elements rhyme: owner-driver, city-state, philosopher-king, hocus-pocus." ([src](https://www.capstoneediting.com/blog/how-to-hyphenate-a-compound-noun))

Also found this: "Hyphenate nouns normally written as two words, when they have a modifier _and when ambiguity would otherwise result_." ([src](https://www.btb.termiumplus.gc.ca/tpv2guides/guides/wrtps/index-fra.html?lang=fra&lettr=indx_catlog_h&page=94llxA6elHZ8.html))

Also this: "Open compounds are typically made up of two nouns that are used together to represent a single idea. “Open” means that there is a space between the two words and no hyphen. Again, a good dictionary is the best place to find out how to treat compounds—if the compound isn’t in the dictionary, that means it should be open." ([src](https://www.grammarly.com/blog/hyphen/))

So my impression is that mostly only "weird" and well-known expressions like "owner-driver" and  "city-state" should have hyphen, in addition to cases where we need to avoid ambiguity.

@JulianNymark What do you think?

Reply:
Yeah, getting to the limits of my grammar specific knowledge on this 😅, I think I (sadly) mostly do hyphen by gut feeling, and that's probably tied to the "weird & well-known" expressions (such as "well-known").